src/SharedMoudule/Components/Validator/Validator.ts

```diff
@@ -0,0 +1,71 @@
+export const emailValidation = {
```

Comment:
I like the idea behind this file, but it's not placed in the right place for it! So i recommend you to read about the common react folder structures.